### PR TITLE
Clean up configs and match comment style

### PR DESCRIPTION
**Pull request by yadudoc**
Updating configs to match in description style.

### CODE REVIEW

File: parsl/configs/kubernetes.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            cores_per_worker=1,
+            max_workers=1,
+            worker_logdir_root='YOUR_WORK_DIR',
+
+            # Address for the pod worker to connect back
+            address=address_by_route(),
+            provider=KubernetesProvider(
+                namespace="default",
+
+                # Docker image url to use for deployment
```
Comment (ZhuozhaoLi) [inline parsl/configs/kubernetes.py]: Better to say `for pods` rather than `for deployment`?

Comment (yadudoc) [reply parsl/configs/kubernetes.py]: Fixed.

**Comment by benclifford** [comment]
need to update `userguide/configuring.rst` which includes some of these by path reference?

**Comment by yadudoc** [comment]
@benclifford Just updated the config references

**Comment by benclifford** [comment]
```
make html
```
...

```
/home/benc/parsl/src/parsl/docs/userguide/configuring.rst:318: WARNING: Include file '/home/benc/parsl/src/parsl/parsl/configs/ec2_htex_single_node.py' not found or reading it failed
```

**Pull request merged**